### ISSUE TITLE
Does not include actual targets of the main project

### ISSUE THREAD

**Issue by beefon** [segiddins/cocoapods-dependencies#18]
The tool is interesting, but is there a way to list deps of the given target inside main project rather than list the deps of all pods all targets in main project refers to? Basically I want to see a tree of deps only for the given target.

**Comment by segiddins** [comment]
that seems possible, but I doubt ill add that functionality myself -- you're more than welcome too if you'd like 👍

**Comment by bpoplauschi** [comment]
@segiddins great tool, thanks for making it open source. Very effective for inspecting your dependencies schema.

Regarding the actual project targets, I also found the need to see them in the graph.
I have altered the project a bit for my usecase, but I'm not sure if other people would be interested in this.
Basically, I'm trying to get all the actual targets from the project as top nodes, instead of the `Podfile` node that is currently at the top of the graph.

I've just used the following changes to the `dependencies.rb` command file.
@segiddins let me if I should create a PR and if we would need a new option.

```ruby
def graphviz_data
  @graphviz ||= begin
    require 'graphviz'
    graph = GraphViz::new(output_file_basename, :type => :digraph)

    unless @podspec
      podfile.target_definitions.values.each do |target|
        target_node = graph.add_node(target.name.to_s)
        if target.dependencies
          target.dependencies.each do |dependency|
            pod_node = graph.add_node(dependency.name.to_s)
            graph.add_edge(target_node, pod_node)
          end
        end
      end
    end

    pod_to_dependencies.each do |pod, dependencies|
      pod_node = graph.add_node(sanitized_pod_name(pod))
      dependencies.each do |dependency|
        dep_node = graph.add_node(sanitized_pod_name(dependency))
        graph.add_edge(pod_node, dep_node)
      end
    end

    graph
  end
end
```

**Comment by segiddins** [comment]
@bpoplauschi looks good! Might make sense to make that an option?

**Comment by bpoplauschi** [comment]
@segiddins That was my question, I think not everyone needs this option, but it would be good to have it available. I can create a PR.

**Comment by bpoplauschi** [comment]
I guess this issue should be closed, since #20 was merged and 1.3.0 release is out.
